upsetplot/plotting.py

```diff
@@ -478,46 +483,80 @@ def plot_intersections(self, ax):
         """Plot bars indicating intersection size
         """
         ax = self._reorient(ax)
-        rects = ax.bar(np.arange(len(self.intersections)), self.intersections,
+        intersections = self.intersections
+        if self.normalize_counts:
+            intersections = 100 * (intersections / intersections.sum())
+        rects = ax.bar(np.arange(len(intersections)), intersections,
                        .5, color=self._facecolor, zorder=10, align='center')
 
         self._label_sizes(ax, rects, 'top' if self._horizontal else 'right')
+        if self.normalize_counts:
+            self._label_sizes(
+                ax, rects, 'bottom', '%d', 0 if self._horizontal else -90,
+                list(self.intersections.values),
+            )
 
         ax.xaxis.set_visible(False)
         for x in ['top', 'bottom', 'right']:
             ax.spines[self._reorient(x)].set_visible(False)
 
         tick_axis = ax.yaxis
         tick_axis.grid(True)
-        ax.set_ylabel('Intersection size')
-
-    def _label_sizes(self, ax, rects, where):
+        ax.set_ylabel(
+            'Intersection {}'.format(
+                'Percentage' if self.normalize_counts else 'Size'
+            )
+        )
+
+    def _label_sizes(self, ax, rects, where, fmt=None, rotation=None,
+                     values=None):
         if not self._show_counts:
             return
-        fmt = '%d' if self._show_counts is True else self._show_counts
+        if not fmt:
+            fmt = '%d' if self._show_counts is True else self._show_counts
+        if values and (len(values) != len(rects)):
```

Comment:
since this should never be violated by user input, might be better as `assert values is None or len(values) == len(rects)`

Reply:
Changed c6914ef4f796f7b4e7a697dfc58f38e6df995798.